### DISCUSSION TITLE
sleuth.db

### DISCUSSION THREAD

**Discussion by gisms** [sepinf-inc/IPED#1618] [Q&A] [answered]
Hi. 

We have a question, when does iped create sleuth.db?
We noticed that in indexing ufdr it is not creating. is that correct?

**Comment by wladimirleite** [comment, marked as answer]
Yes, that is the expected behavior.
Sleuth.db is created for evidences handled by TSK (The Sleuth Kit), which are forensic images like DD, 001 and E01 files.
Folders and UFDRs are read directly by IPED.

**Comment by gisms** [comment]
thak u a lot!!!
